brain/compartmentReportView.cpp

```diff
@@ -120,9 +56,22 @@ const CompartmentReportMapping& CompartmentReportView::getMapping() const
 }
 
 std::future<CompartmentReportFrame> CompartmentReportView::load(
-    const float timestamp)
+    double timestamp)
 {
+    const double start = _impl->report->getStartTime();
+    const double end = _impl->report->getEndTime();
+
+    if (timestamp < start || timestamp >= end)
+        throw std::logic_error("Invalid timestamp");
+
+    const double timestep = _impl->report->getTimestep();
+
+    size_t frameIndex = std::llround((timestamp - start) / timestep);
```

Comment:
Actually, some tests are failing when using floor. 
I also would prefer to use the same thing as the low level CR (I hesitated to use any ugly extern before duplicating the code).
Either we change both or nothing.